src/main/java/hudson/plugins/mercurial/HgExe.java

```diff
@@ -346,6 +348,40 @@ public ProcStarter run(ArgumentListBuilder args) {
     }
 
     /**
+     * Gets the branch name of given revision number or of the current workspace.
+     * @param rev the revision to identify; defaults to current working copy
+     */
+    public @CheckForNull String branch(FilePath repository, @Nullable String rev) throws IOException, InterruptedException {
```

Comment:
Use `@CheckForNull`. (`@Nullable` means something much more complicated and rarely wanted.)